### DISCUSSION TITLE
Getting parse error while compiling fundMe in lesson 4

### DISCUSSION THREAD

**Discussion by elec7rik** [Cyfrin/foundry-full-course-cu#706] [Q&A] [answered]
![Screenshot from 2023-08-24 20-59-24](https://github.com/Cyfrin/foundry-full-course-f23/assets/89316185/acf16ce4-9c1a-42b0-a000-144d9916b7ad)

**Comment by Naman1729** [comment]
Hey @tend0pain , Try to reload your remix. It compiles successfully on my end.

**Reply by elec7rik** [reply]
I did, it's not working. Is your code exactly the same? 
It was working earlier then I tried to import the latestRoundData() function by writing in the {} and was trying to change the path, it didn't work. Then I wrote this code back and it's not working either.
Can you share your code? I'll try to match.

**Comment by theirrationalone** [comment, marked as answer]
# COPY YOUR ALL CODE IN A SAFE PLACE i.e., IN NOTEPAD BEFORE DOING THIS.
on remix window press `ctrl + shift + i` and reach here as given below.... After that, Please click and press `delete` key on your keyboard and then reload....

![remix fix](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/4de6845d-b7a9-4cd4-9c6e-dcb4cce3c21d)

![remix fix 2](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/a8aa7f3e-defd-4a66-bb1b-7276ce0ef560)

**Reply by elec7rik** [reply]
it just worked, thanks bud. any guess why this problem arises?

**Reply by theirrationalone** [reply]
Incomplete fetched information was got stored into browser's local storage that's why your browser was only using local storage information for speeding up your interactions... We just removed local data that forced browser to collect Or retrieve new complete data.. 

Nice working with you @tend0pain , Genuinely appreciating your curiosity! 👏👏👏
